docs/source/quick_start.rst

```diff
@@ -0,0 +1,199 @@
+   -DKokkos_ENABLE_SERIAL=ON \
+   -DCMAKE_BUILD_TYPE=Release ..
+
+
+CUDA (CPU and GPU)
+~~~~~~~~~~~~~~~~~~
+
+::
+
+  cmake \
+    -DKokkos_ARCH_NATIVE=ON \
+    -DKokkos_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE=OFF \
+    -DKokkos_ENABLE_CUDA=ON \
+    -DKokkos_ENABLE_SERIAL=ON \
+    -DCMAKE_BUILD_TYPE=Release ..
+
+
+HIP (CPU and GPU)
+~~~~~~~~~~~~~~~~~
+::
+
+  cmake \
+    -DKokkos_ARCH_NATIVE=ON \
+    -DKokkos_ENABLE_HIP=ON \
+    -DKokkos_ENABLE_SERIAL=ON \
+    -DCMAKE_CXX_COMPILER=hipcc \
+    -DCMAKE_BUILD_TYPE=Release ..
+
+
+----------------------------------------------
+Basic Configuration Examples for Spack Builds
+----------------------------------------------
+
+|br|
+Diverse Kokkos variants can be built / installed via Spack.  You will need to select a variant for the desired backend, and appropriate GPU microarchitecture.  To explore the range of variants for a package, ``spack info kokkos``, ``spack info trilinos``, etc.  Please see basic `Spack installation  <https://spack.readthedocs.io/en/latest/getting_started.html>`_ instructions if you're new to the package manager.
+|br|
+|br|
+
+
+.. note::
+
+  Before installing, you can ``spack spec``  variants to verify the build type.
+
+Serial
+~~~~~~~
+
+::
+
+  spack spec kokkos@4.2 %gcc@10.3.0 +serial cxxstd=20
+
+OpenMP
+~~~~~~
+
+::
+
+  spack spec kokkos@4.2 %gcc@10.3.0 +openmp cxxstd=20
+
+
+CUDA
+~~~~
+
+:: 
+  
+  spack spec / install kokkos@4.2 %gcc@12.2.0 +cuda cuda_arch=70 cxxstd=20 +cuda_relocatable_device_code
+
+
+HIP
+~~~
+
+::
+
+  spack spec / install kokkos@4.2 %gcc@12.2.0 +rocm amdgpu_target=gfx90a cxxstd=20
+
+
+-------------------------------------------
+Building and Linking a Kokkos "Hello World"
+-------------------------------------------
+
+.. note::
+
+  You will need to set ``Kokkos_ROOT``, and also the root directory for you target backend SDK (i.e., ``CUDA_ROOT``, ``ROCM_PATH``).  Please see `Build, Install and Use <https://kokkos.org/kokkos-core-wiki/building.html>`_ for additional details.
+
+|br|
+::
+
+  git clone https://github.com/ajpowelsnl/View
+  cd View
+  mkdir build
+  cd build
+  cmake ../
+
+
+-------------
+Get Involved!
+-------------
+
+Joining the Kokkos Community
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Participate in the lively virtual discussion on the `Kokkos Slack Channel <https://kokkosteam.slack.com>`_.  Here are `sign up details <https://kokkos.org/kokkos-core-wiki/faq.html#faq>`_.
+|br|
+
+Acclerating learning
+~~~~~~~~~~~~~~~~~~~~
+
+Take a deeper dive into Kokkos with over 16 hours of `Tutorials <https://github.com/kokkos/kokkos-tutorials>`_ and `Recorded Lectures <https://github.com/kokkos/kokkos-tutorials/wiki/Kokkos-Lecture-Series>`_.  For in-house workshops and training, please get in touch via Slack (below).
+|br|
+
+Coming to Release Demos
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You have a standing invitation to attend our approximately quarterly release demos to hear what's new, what's gone, and where we're heading in the future.
+|br|
```

Comment:
```suggestion
Coming to Release Demos
~~~~~~~~~~~~~~~~~~~~~~~

You have a standing invitation to attend our approximately quarterly release demos to hear what's new, what's gone, and where we're heading in the future.
```

Reply:
I introduced the `|br|` for readability, and would like to leave them there.